turborepo-tests/integration/tests/run/single-package/run-yarn.t

```diff
@@ -1,17 +1,11 @@
 Setup
-  $ . ${TESTDIR}/../../../../helpers/setup.sh
-  $ . ${TESTDIR}/../../_helpers/setup_monorepo.sh $(pwd) single_package "yarn@1.22.17"
-  $ rm -rf package-lock.json || true # exists because of setup.sh script above
-  $ yarn install > /dev/null 2>&1
-Run dos2unix on Windows for newly generated lockfile.
-  $ if [[ "$OSTYPE" == "msys" ]]; then dos2unix --quiet yarn.lock; fi
-  $ git commit --quiet -am "Update lockfile" # clean git state
+  $ . ${TESTDIR}/../../../../helpers/setup_integration_test.sh $(pwd) single_package "yarn@1.22.17"
 
 Check
   $ ${TURBO} run build
   \xe2\x80\xa2 Running build (esc)
   \xe2\x80\xa2 Remote caching disabled (esc)
-  build: cache miss, executing 551563378a0c0a7e
+  build: cache miss, executing c1d6153f4a6d83b5
```

Comment:
This changed because with the `install_deps.sh` script has an additional feature where it sets a custom cache dir and adds it to a local gitignore file. The change to gitignore updated the task hash

```
yarn install --cache-folder="$PWD/.yarn-cache" > /dev/null 2>&1
echo ".yarn-cache" >> .gitignore
```